Casks/gu-base.rb

```diff
@@ -15,37 +15,21 @@
   depends_on formula:  'bash-completion'
   depends_on formula:  'coreutils'
   depends_on formula:  'git'
-  depends_on formula:  'hub'
   depends_on formula:  'gnupg'
   depends_on formula:  'htop'
-  depends_on formula:  'mtr'
   depends_on formula:  'tree'
   depends_on formula:  'watch'
   depends_on formula:  'awscli'
   depends_on formula:  'nginx'
 
   # dev langs
-  depends_on cask:     'adoptopenjdk/openjdk/adoptopenjdk8'
   depends_on cask:     'gu-scala'
-  depends_on formula:  'node'
-  depends_on formula:  'yarn'
```

Comment:
Prefer _not_ to install node and yarn globally. Instead recommend using `asdf` or similar.